server/controller/src/main/java/ai/starwhale/mlops/common/RegExps.java

```diff
@@ -16,8 +16,6 @@
 
 package ai.starwhale.mlops.common;
 
-import java.util.regex.Pattern;
-
 public class RegExps {
 
     public static final String USER_NAME_REGEX = "^[a-zA-Z][a-zA-Z\\d_-]{3,32}$";
```

Comment:
How about moving `RegExps.USER_NAME_REGEX` to `UserRequest` where this field is used only?

Reply:
has moved regexes to service classes